pages/dashboard.tsx

```diff
@@ -136,81 +126,50 @@ const Dashboard: NextPage = () => {
           </div>
           <div className="md:flex justify-center">
             <div className="flex-1">
-              <h1 className="text-3xl font-bold tracking-tight text-gray-800 mb-4 mt-8 text-center">
-                Upcoming Events &#8250;
+              <h1 className="text-3xl font-bold tracking-tight text-gray-800 my-4 text-center">
+                {events.length > 0 ? `What's Next ›` : `More events coming soon!`}
```

Comment:
If there are no events, can we leave the header as "What's next" and add a `<p>` tag below saying "More events coming soon"? Changing the header feels weird + more text might fill the whitespace better